### ISSUE TITLE
[Alchemy] Guardian and Battle Elixirs stacking, and Flasks stacking

### ISSUE THREAD

**Issue by DaRealBoss** [ccshiro/cc-buglist#838]
You can currently have as many guardian and battle elixirs up as you want. They all stack (assuming they are of different kind). A better version of the same elixir does replace the lesser one of the same kind which is blizzlike (ex. elixir of superior defense replaces elixir of greater defense when used). As far as I know, you are only supposed to be able to use 1 guardian elixir and 1 battle elixir at any time, or a flask.

Using a flask cancelled/replaced all the battle and guardian elixirs I had up. However it seems that you can also stack all kinds of flasks together.

In conclusion, you can't have flasks and battle/guardian elixirs up at the same time, which is blizzlike. But you can stack flasks with other flasks, and battle/guardian elixirs with others which is not blizzlike

**Comment by Kall-DestinyAwaits** [comment]
Before patch 2.1 players could stack 1xFlask+4xElixirs and other consumables. All Nihilum members for example on Gruul used:
Elixir of Mastery+Elixir of Major Firepower+Elixir of Vitality (removed)+Adept's Elixir+Flask of Supreme Power

On this Video (prenerfed Gruul by Forte) we can see many elixirs and flask on his buff bar addon.
https://www.youtube.com/watch?v=u1LOAT96vEs

Sources:
http://manaflask.com/en/articles/eoy-s-exploits-the-burning-crusade
http://www.wow-one.com/forum/topic/38751-elixir-flask-stacking

So its not bug - its work like before 2.1 patch
The only bug its that you can use more then 1 flask in the same time
